src/Plugin/Purge/Purger/AkamaiPurger.php

```diff
@@ -93,4 +85,11 @@ public function invalidate(array $invalidations) {
     $this->client->purgeUrls($urls_to_clear);
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function hasRuntimeMeasurement() {
+    return TRUE;
+  }
+
```

Comment:
Ok, so looking at Purge, it appears that if we report this as true we need to implement a RuntimeMeasurementInterface.